### PR TITLE
[FIX] Variable rendering problem on Import recent history page

### LINKED ISSUE #15996: Incorrect rendering of variable in the ui
### Description:
In the recent import history, the import type variable is not rendering correctly as the underscores are appearing . 

### Steps to reproduce:

1. Click on Administration
2. Go to recent import history
3. The import type variable is not rendered correctly

### Expected behavior:
It should render correctly so that translation could be possible
### Actual behavior:

image

### Server Setup Information:

- Version of Rocket.Chat Server: 3.0.0 develop
- Operating System:  ubuntu

### Are you working on this
Yes

### PR DESCRIPTION
**Pull request by ritwizsinha**
Closes #15996 

image
Here is the screenshot of the fixed variable rendering problem

### CODE REVIEW

**Comment by ritwizsinha** [comment]
Please Review

**Comment by rodrigok** [comment]
@ritwizsinha thanks for your contribution.

Can you move the translation method to the template so it will cover all the other importers as well?

Are you ok to do this change?

**Comment by ritwizsinha** [comment]
@rodrigok sorry if I am misunderstanding but the instance.history.curValue that I have looped over already contains all the other importer translations so why do we need to move it to template? Of course I am OK to do this change

**Comment by rodrigok** [comment]
Sorry. You are right, I was thinking on the other PR. The case here is that you remove the original value of the property type, if it’s necessary to be used elsewhere it will generate a problem, that’s why we usually translate things on the template or inside a helper that return the translation for that case.

**Comment by ritwizsinha** [comment]
@rodrigok Thanks for the input I would change my pr accordingly to make changes in the template

**Comment by ritwizsinha** [comment]
@rodrigok please review

**Comment by ritwizsinha** [comment]
@rodrigok please review

**Comment by ritwizsinha** [comment]
Please review

File: app/importer/client/admin/adminImportHistory.js [hunk 1]
```diff
@@ -174,6 +174,10 @@ Template.adminImportHistory.helpers({
 		const date = new Date(this.msg.ts);
 		return date.toLocaleString();
 	},
+	translate(type) {
```
Comment (gabriellsh) [inline app/importer/client/admin/adminImportHistory.js]: delete this helper

File: app/importer/client/admin/adminImportHistory.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 								<h1>{{ statusMessage}}</h1>
 
 								<span style="font-weight: 600;">{{_ "Import_Type"}}:</span>
-								<span>{{ type}} [{{ importerKey}}]</span>
+								<span>{{translate type}} [{{ importerKey}}]</span>
```
Comment (gabriellsh) [inline app/importer/client/admin/adminImportHistory.html]: change to 
`<span>{{_ type}} [{{ importerKey}}]</span>`
the "_" symbol is a global helper that does the same thing as the one you created.

**Comment by ritwizsinha** [comment]
@gabriellsh I have made the changes please review

File: app/importer/client/admin/adminImportHistory.js [hunk 3]
```diff
@@ -174,6 +174,7 @@ Template.adminImportHistory.helpers({
 		const date = new Date(this.msg.ts);
 		return date.toLocaleString();
 	},
+
```
Comment (gabriellsh) [inline app/importer/client/admin/adminImportHistory.js]: You left a whitespace here.

Comment (ritwizsinha) [reply app/importer/client/admin/adminImportHistory.js]: Made the change

**Comment by gabriellsh** [comment]
@ritwizsinha Could you solve conflicts please?

**Comment by ritwizsinha** [comment]
@gabriellsh the resolve conflicts button is not enabled for me, I can't change it.

**Comment by gabriellsh** [comment]
Yes, you have to checkout to your branch and do it manually.

**Comment by ritwizsinha** [comment]
I am not able to find that file as adminImportHistory.html was deleted.

**Comment by gabriellsh** [comment]
I just pushed the changes :) You can check it im modified files now.

**Comment by ritwizsinha** [comment]
Ok so the changes are done

**Pull request merged**